### PR TITLE
feat(ctb): add getting-started network

### PR DESCRIPTION
**Pull request by smartcontracts**
**Description**
Adds the getting-started network and configuration for the OP Stack Getting Started guide.

### CODE REVIEW

File: packages/contracts-bedrock/hardhat.config.ts [hunk 1]
```diff
@@ -84,6 +84,12 @@ const config: HardhatUserConfig = {
       accounts: [process.env.PRIVATE_KEY_DEPLOYER || ethers.constants.HashZero],
       live: true,
     },
+    'getting-started': {
+      chainId: 5,
+      url: process.env.L1_RPC || '',
+      accounts: [process.env.PRIVATE_KEY_DEPLOYER || ethers.constants.HashZero],
+      live: true,
+    },
```
Comment (maurelian) [inline packages/contracts-bedrock/hardhat.config.ts:92]: What is this? 
Where will it be documented?
Should it be on goerli?

**Review by maurelian — APPROVED** [review]
Approving pending resolution of comment

File: packages/contracts-bedrock/hardhat.config.ts (hunk 1 above)
Comment (smartcontracts) [reply packages/contracts-bedrock/hardhat.config.ts:92]: It's the network that will be used for the OP Stack documentation tutorial. It should be on Goerli, yes.

**Comment by smartcontracts** [comment]
Moving to draft while I tweak some parameters

**Pull request merged**